### DISCUSSION TITLE
mojo中文文档，以及用户交流群（微信、qq、TG）

### DISCUSSION THREAD

**Discussion by shadowqcom** [modular/modular#748] [General]
⚡欢迎各位开发者加群 摸鱼 交流。
 mojo中文网 [mojocn.org](https://mojocn.org/)

用户群：[点击前往](https://mojocn.org/docs/chat.html)

**Comment by gryznar** [comment]
Please use english to help people understand

**Reply by shadowqcom** [reply]
我们使用中文，并不妨碍你使用英文。中文文化包容且开放，中文用户积极学习新技术，我们不吝与世界共享。

**Reply by LiMao00** [reply]
就是，什么玩意还强制人使用指定语言讨论
